pom.xml

```diff
@@ -1164,7 +1164,27 @@
       <dependency>
         <groupId>io.dropwizard.metrics</groupId>
         <artifactId>metrics-core</artifactId>
-        <version>4.0.2</version>
+        <version>4.1.0-rc0</version>
```

Comment:
define version as a property and avoid using `rc` version unless it provides a significant benefit.